packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -55,18 +66,71 @@ async function getConsolidatedAndSendToCx({
   resources,
   dateFrom,
   dateTo,
+  conversionType,
 }: {
   patient: Pick<Patient, "id" | "cxId">;
   resources?: ResourceTypeForConsolidation[];
   dateFrom?: string;
   dateTo?: string;
+  conversionType?: ConsolidationConversionType;
 }): Promise<void> {
   const { log } = Util.out(
     `getConsolidatedAndSendToCx - cxId ${patient.cxId}, patientId ${patient.id}`
   );
   const filters = { resources: resources ? resources.join(", ") : undefined, dateFrom, dateTo };
   try {
-    const bundle = await getConsolidatedPatientData({ patient, resources, dateFrom, dateTo });
+    let bundle = await getConsolidatedPatientData({ patient, resources, dateFrom, dateTo });
+
+    if (conversionType === "html" || conversionType === "pdf") {
+      const fhir = makeFhirApi(patient.cxId);
+
+      const fhirPatient = await fhir.readResource("Patient", patient.id);
+
+      const bundleWithPatient: Bundle<Resource> = {
+        ...bundle,
+        total: (bundle.total ?? 0) + 1,
+        entry: [
+          {
+            resource: fhirPatient,
+          },
+          ...(bundle.entry ?? []),
+        ],
+      };
+
+      const url = await convertFHIRBundleToMedicalRecord({
+        bundle: bundleWithPatient,
+        patient,
+        resources,
+        dateFrom,
+        dateTo,
+        conversionType,
+      });
+
+      bundle = {
+        resourceType: "Bundle",
+        total: 1,
```

Comment:
Shouldn't we send the consolidated data along with the result of the conversion?

Reply:
My thought with this was if they are expecting an html or pdf file to only return that. If we return the whole result of the bundle that may be confusing